code/modules/client/preferences_gear.dm

```diff
@@ -415,6 +415,33 @@ var/global/list/gear_datums_by_name = list()
 	display_name = "Camera"
 	path = /obj/item/device/camera
 
+/datum/gear/toy/mags
+	cost = 1
+
+/datum/gear/toy/mags/magazinedirty
```

Comment:
snake case all these type paths

snake_case_all_these_type_paths <--- example

Reply:
I'm not sure what you're asking here. The other typepaths aren't snake-cased, what causes these ones the need to be?

Do you want me to change `/datum/gear/toy/mags/magazinedirty/` into `/datum_gear_toy_mags_magazinedirty/` or something? The example you gave is a little vague.

Reply:
/datum/gear/toy/mags/magazine_dirty

Reply:
Should be fine now?

Reply:
Beautiful!